src/Views/Email/email_2fa_email.php

```diff
@@ -1 +1,19 @@
-<p><?= lang('Auth.email2FAMailBody') ?> <b><?= $code ?></b></p>
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
+<html xmlns="http://www.w3.org/1999/xhtml" xmlns="http://www.w3.org/1999/xhtml">
+
+<head>
+<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title><?= lang('Auth.email2FASubject') ?></title>
+</head>
+
+<body>
+    <p><?= lang('Auth.email2FAMailBody') ?></p>
+    <div style="text-align: center"><h1><?= $code ?></h1></div>
+    <b><?= lang('Auth.emailInfo') ?></b>
+    <p><?= lang('Auth.emailIpAddress') ?> <?= $ipAddress ?></p>
+    <p><?= lang('Auth.emailDevice') ?> <?= $userAgent ?></p>
+    <p><?= lang('Auth.emailDate') ?> <?= $date ?></p>
```

Comment:
Any variable in HTML templates whose value is a text string rather than an HTML string must be escaped.


```suggestion
    <div style="text-align: center"><h1><?= esc($code) ?></h1></div>
    <b><?= lang('Auth.emailInfo') ?></b>
    <p><?= lang('Auth.emailIpAddress') ?> <?= esc($ipAddress) ?></p>
    <p><?= lang('Auth.emailDevice') ?> <?= esc($userAgent) ?></p>
    <p><?= lang('Auth.emailDate') ?> <?= esc($date) ?></p>
```